src/mesh/mesh.hpp

```diff
@@ -212,6 +212,11 @@ class Mesh {
   // size of default MeshBlockPacks
   int default_pack_size_;
 
+#ifdef MPI_PARALLEL
+  // Global map of MPI comms for separate variables
+  std::map<std::string, MPI_Comm> mpi_comm_map_;
```

Comment:
Maybe should be `unordered_map`, although I doubt it matters.

Reply:
I had a misplaced `const` that did nothing and I am about to fix. Will update this at the same time.